client/src/components/_common/Section/SectionMessages.jsx

```diff
@@ -48,20 +48,28 @@ import './SectionMessages.css';
 function SectionMessages({
   children,
   className,
-  introMessageName,
+  messageComponentName,
   introMessageText,
 }) {
-  const introMessageContent = introMessageText || MESSAGES[introMessageName];
+  const introMessageContent =
+    introMessageText || MESSAGES[messageComponentName];
   const introMessage = introMessageContent && (
     /* FAQ: Alternate message name allows tracking custom message dismissal */
-    <IntroMessage messageName={introMessageName || introMessageText} canDismiss>
+    <IntroMessage
+      messageComponentName={messageComponentName || introMessageText}
+      canDismiss
+    >
       {introMessageContent}
     </IntroMessage>
   );
   const hasMessage =
-    isKnownIntroMessage(introMessageName) || children.length > 0;
+    isKnownIntroMessage(messageComponentName) || children.length > 0;
   const hasMessageClass = 'has-message';
 
+  const customMessage = (
+    <CustomMessage messageComponentName={messageComponentName}></CustomMessage>
+  );
+
```

Comment:
@duckonomy I'm a little confused here. It seems both the `IntroMessage` and `CustomMessage` components are relying on the same `messageComponentName` parameter - is this preventing both an intro message and a custom message? My initial feeling is it would be clearer for `IntroMessage` to ingest `introMessageName`, and `CustomMessage` to ingest `customMessageName`

Reply:
@rstijerina So a separate prop?

Reply:
That's what I was thinking. Unless you think I'm missing something?